gui.py

```diff
@@ -29,10 +29,7 @@ def create_cell(self, sudoku_frame, row_id, col_id, value):
         """
         digits = ["1", "2", "3", "4", "5", "6", "7", "8", "9", " "]
         tkvar = tk.StringVar(self.root)
-        if value is None:
-            value = " "
-        else:
-            value = str(value)
+        value = " " if value is None else str(value)
```

Comment:
I don't think this one liner improves performance.
But it definitely reduces readability.
In my opinion this change should not be applied.

Reply:
Personal preference i guess. Although the only reason I considered these was because I found even more complex one-liners and thought you are not against reducing line-count for slightly less readable code. Will revert those back then

Reply:
In some cases, one liners are more efficient (for example - list comprehension).

Reply:
Interesting, wouldn't have thought that it could be more efficient